be/src/http/http_common.h

```diff
@@ -35,7 +35,7 @@ static const std::string HTTP_PARTITIONS = "partitions";
 static const std::string HTTP_TEMP_PARTITIONS = "temporary_partitions";
 static const std::string HTTP_NEGATIVE = "negative";
 static const std::string HTTP_STRICT_MODE = "strict_mode";
-static const std::string HTTP_TIMEZONE = "timezone";
```

Comment:
We should assure backward compatibility. I suggest we just add a new `time_zone` and users can set through both `time_zone` and `timezone`